includes/wikia/GlobalFunctions.php

```diff
@@ -117,7 +117,7 @@ function wfReplaceImageServer( $url, $timestamp = false ) {
 	// FIXME: This needs to be removed. It should be encapsulated in the URL generation.
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Take the required action to fix the issue indicated by this "FIXME" comment. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS1134)